securedrop/manage.py

```diff
@@ -66,9 +66,13 @@ def add_journalist():  # pragma: no cover
     return _add_user()
 
 
+def get_username():  # pragma: no cover
+    return raw_input('Username: ')
+
+
 def _add_user(is_admin=False):  # pragma: no cover
     while True:
-        username = raw_input('Username: ')
+        username = get_username()
```

Comment:
Also, since we're further complicating this one function (`_add_user`), this would also be a perfect opportunity to add tests for this one function (partially addresses #1982).